bundles/admin/admin-announcements/publisher/AnnouncementsTool.js

```diff
@@ -184,11 +185,11 @@ Oskari.clazz.define('Oskari.admin.bundle.admin-announcements.publisher.Announcem
 
             // Announcement is selected
             content.find('input[name=announcement]').on('change', function () {
-                var announcement = me.announcements.find(item => item.title === jQuery(this).val());
+                var announcement = me.announcements.find(item => item.locale[me.lang].name === jQuery(this).val());
```

Comment:
Is there a reason we can't use announcement.id here?

Reply:
I think it should be saved in only one place and that would be the plugin config, but it's ok for now as this is as it has been.